William Rudder/week4/books/views/bookcover.erb

```diff
@@ -0,0 +1,9 @@
+<p>the book you searched for: </p>
+<p><%= params[:book] %></p>
+
+
+
+<%=@imageurl%>
+
+<img src="@imageurl">
```

Comment:
So close!
You need this:
``` ruby 
<img src="<%= @imageurl %>">
```
You just need the '<%= %>' around @imageurl to indicate that its ruby code.